extensions/open-remote-ssh/src/authResolver.ts

```diff
@@ -244,13 +244,15 @@ export class RemoteSSHResolver implements vscode.RemoteAuthorityResolver, vscode
 				if (context.resolveAttempt === 1) {
 					this.logger.show();
 
-					const closeRemote = 'Close Remote';
-					const retry = 'Retry';
-					const result = await vscode.window.showErrorMessage(`Could not establish connection to "${sshDest.hostname}"`, { modal: true }, closeRemote, retry);
-					if (result === closeRemote) {
-						await vscode.commands.executeCommand('workbench.action.remote.close');
-					} else if (result === retry) {
-						await vscode.commands.executeCommand('workbench.action.reloadWindow');
+					if (!(e instanceof ServerInstallError)) {
+						const closeRemote = 'Close Remote';
+						const retry = 'Retry';
+						const result = await vscode.window.showErrorMessage(`Could not establish connection to "${sshDest.hostname}"`, { modal: true }, closeRemote, retry);
+						if (result === closeRemote) {
+							await vscode.commands.executeCommand('workbench.action.remote.close');
+						} else if (result === retry) {
+							await vscode.commands.executeCommand('workbench.action.reloadWindow');
+						}
 					}
```

Comment:
I figured we don't want the scary "could not connect to host" error whenever the server install script failed, because that means it _could_ connect to the host. And the error will be shown below.

Reply:
Yeah, I think this is one of the things that has confused people!